tests/cli/test_import_command.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from mock import patch
+import yaml
+from pathlib import Path
+
+from click.testing import CliRunner
+
+from mutacc_auto.cli.root import cli
+
+INPUTS_LIST = [{'input_file': 'path_to_file', 'padding':600}]*2
```

Comment:
What does "*2" do? A comment would be nice

Reply:
600 seems like a constant. Could you use named constants to make the code easier to understand. For instance why 600 and not 300?

Reply:
"*2" just duplicates the elements in the list, so test is run with a list of two elements (although identical). Can make 'padding' a constant, Although here the value has no actual importance, this is just to test the function.